core/contextmenu_items.ts

```diff
@@ -554,6 +557,108 @@ export function registerHelp() {
   ContextMenuRegistry.registry.register(helpOption);
 }
 
+/** Registers an option for deleting a workspace comment. */
+export function registerCommentDelete() {
+  const deleteOption: RegistryItem = {
+    // TODO: Before merging, I had to make this a function because I don't think
+    //   the messages were loaded in time. Is that a larger bug?
+    displayText: () => Msg['REMOVE_COMMENT'],
+    preconditionFn(scope: Scope) {
+      return scope.comment?.isDeletable() ? 'enabled' : 'hidden';
+    },
+    callback(scope: Scope) {
+      eventUtils.setGroup(true);
+      scope.comment?.dispose();
+      eventUtils.setGroup(false);
+    },
+    scopeType: ContextMenuRegistry.ScopeType.COMMENT,
+    id: 'commentDelete',
+    weight: 6,
+  };
+  ContextMenuRegistry.registry.register(deleteOption);
+}
+
+/** Registers an option for duplicating a worksapce comment. */
```

Comment:
Nit: workspace

Reply:
Thank you! I have typo'ed this so much my IDE actually autocompletes to the typo'ed version :P